### ISSUE TITLE
Changing the name 

### ISSUE THREAD

**Issue by vladtyum** [hneemann/Digital#151]
I'm sure it sounds a bit too extreme, but please give it a short consideration. 
As a guy who really loves your SW, I'd like to put my files on web, and be able to search and download them online. Yet googling "digital" gives us huge noise, everything but not what we need. 
I'd like to suggest slightly change the name, probably add couple letters after the name something like  DigitalHW,  DigitalSM, DigitalHWSM DIgital[whatever] just to make it easier for us to be able to search it as "Logism" and build community of enthusiasts around it. 
Much love.
Vlad

**Comment by xotmatrix** [comment]
I've resisted posting an issue like this but I agree 100%. "Digital" is unsearchable. I once lost track of this delightful software and it took me ages to find it again.

**Comment by sgparry** [comment]
HNDigital in honour of Helmut? He deserves maximum credit in the name.

**Comment by LarsiParsii** [comment]
A great simulator, but I agree. I wouldn't have found this simulator if it hadn't been for one of my teachers, as it's impossible to search up on the web. :/

**Comment by kal9001** [comment]
Yes. I have literally just had this problem.
I found this software just today while trying to figure out how to get logisim-evolution working with GHDL, and someone suggested this project. but after closing a tab it was next to impossible to re-find it without digging through my history.
Googling "digital" quite rightly gives extremely broad results, even trying to qualify it with "simulator" and other such things doesn't help at all.

**Comment by nevercast** [comment]
Same issue here, me and a friend of mine have built many the CPU in Logisim and when we discovered this software we were overjoyed, but when Uni came around we lost it. GoogleFu "Digital Java Logisim but better" didn't help. Glad I've got it Starred now though!

**Comment by fdedinec** [comment]
In my experience, google Digital Logisim, which works but is too much advertising for logisim.

**Comment by vladtyum** [comment]
@hneemann what's your opinion on that, mister ?
Many thanks for your commitment and making the project going 🖖

**Comment by hneemann** [comment]
@vladtyum 
I'm thinking about "DigitalCS" which stands for Digital Circuit Simulator.

**Comment by augu5te** [comment]
+1, I redicovered Digital (I've forgotten it, not Logisim).
IMHO, having "Sim" in the name is more explicit, CS equal implicitly Computer Science.
Great project in any case !

**Comment by shawty** [comment]
As a longtime Logisim user, and now "Digital" user, I concur.  I find myself having to use "github digital" or "hnee digital" (I can never remember full name) just to find it when looking for the github page.

Since your essentially "Logisim" the next generation, you could maybe also pay homage to Logisim somwhere along the line too.

**Comment by hrobeers** [comment]
Another important reason to change the name is that it is impossible to search for component libraries online.

e.g. I'm looking for an SPI serial EEPROM, but there is no obvious search term to figure out whether someone might be sharing one online.

**Comment by harwoodr** [comment]
The Esperanto word for "digitial" is "cifereca" - relatively unique on the internet.

**Comment by hneemann** [comment]
@harwoodr 
My current favorite is "DigitaSi" because of the remark from @augu5te.

**Comment by JarrettBillingsley** [comment]
I was gonna suggest "Digisim" to make it an homage to "Logisim", but it seems there's already some software named that :(

**Comment by fuomag9** [comment]
I agree with the issue, if "digital github" would not have yielded results I'd never have found it

**Comment by charlespax** [comment]
I came here to file this exact issue. Digital is pretty easy to use, so it feels like a simple simulator. What about calling it "simsim?" Google returns fewer than fifty results.

![Screenshot from 2020-07-26 14-48-05](https://user-images.githubusercontent.com/281642/88486904-0c109200-cf4f-11ea-85df-d11ee05fb4a4.png)

**Comment by jwbruce** [comment]
I will have to pile on here too.  Digital is a great little tool.  My students learn a lot about combinational and sequential design by using it in my class.  However, the name "Digital" makes the tool very hard to locate on the internet, find any other references to tutorials or examples by 3rd parties, AND when discussing Digital in my digital systems class about digital design and digital logic gates it can get very confusing.  If there is a desire to not to change the name too much.... how about "Digitalis"?  Really anything other than a word that literally is an adjective modifier found in 50% of online subject matter.

**Comment by jay20162016** [comment]
This is unoriginal, but what about "Logisim Digital" as a name?

**Comment by TheWildJarvi** [comment]
Okay, I have heard numerous complaints about the name being unsearchable on google. My suggestion, "Digisim".

**Comment by sgparry** [comment]
Is three years enough for time this discussion?
@jwbruce Digitalis certainly gets the heat pounding...
@hneemann did you deliberately leave out the l from DigitaSi?

**Comment by peterb12** [comment]
FYI I've been doing a YouTube series using Digital and "Where can I find this?" is the number one question people have. They absolutely are unable to locate the software by typing "digital" into google. A name change to something searchable would really help this software's adoption, IMO.

**Comment by charles-fox** [comment]
I came here from logic-evolution's page, which is still the only way I can find it!    Currently looking at both and comparing, logic-evolution seems to have the bigger community around it, which is important, in part due to this naming issue?

**Comment by trayres** [comment]
This project is amazing and deserves a name that is more discoverable.

**Comment by mspeculatrix** [comment]
Just want to add my +1 (the first of 2022) for a name change.

**Comment by Rishav-mngo** [comment]
The name is really hindering the popularity. Please consider changing the name.  +1

**Comment by NotCoolKiwi** [comment]
Amazing piece of software but can't find anything on google. Possibly consider changing the name. Another solution could be to create some kind of database or built-in tool where people can upload their own chips and such. This way, we may be able to find what we are looking for easier. As the naming may also be hindering the popularity, maybe consider doing both. Thank you so much!

**Comment by CMDR-Piboy314** [comment]
+1

**Comment by sgparry** [comment]
I think we need to up-vote on specific names, rather than just "we want a new name" so: HNDigital + 1...

On 31 July 2022 20:01:21 BST, CMDR-Piboy314 ***@***.***> wrote:
>+1
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/hneemann/Digital/issues/151#issuecomment-1200480585
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by shawty** [comment]
+1 for HNDigital - works for me.

**Comment by CMDR-Piboy314** [comment]
+1 for HNDigital

**Comment by rejahtavi** [comment]
HNDigital looks good to me, but if you're against including your initials, or want something more creative, I suggest "Aristottl".

It's a mashup of TTL and Aristotle, the Greek philosopher known for his work on early formal logic. As a bonus, Google finds even less for it than HNDigital.

**Comment by sgparry** [comment]
I still think HNDigital works well as a name; Another possiblity: HEDSim; Highly Efficient Digital Simulator. 

On 31 July 2022 20:01:21 BST, CMDR-Piboy314 ***@***.***> wrote:
>+1
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/hneemann/Digital/issues/151#issuecomment-1200480585
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by ekm456** [comment]
Please change the name to make it searchable.

**Comment by charlespax** [comment]
This has been an issue sine 2018. At what point should someone fork Digital, rename it, and consider Digital as the upstream of whatever is the new name?

**Comment by gresolio** [comment]
Google stubbornly refused to find this project, giving many irrelevant links...
A successful search query for me is "logisim alternative digital", which finally led me to the post with the correct link:
https://www.reddit.com/r/logisim/comments/kv9qkg/digital_a_logisim_alternative/

**Comment by chuckhoffmann** [comment]
The search I use is "Neemann Digital". Seems to work pretty well, and I just have to remember Herr Professor Neemann's last name. Maybe just formalize the name of the software as "Neeman Digital"?

**Comment by shawty** [comment]
@chuckhoffmann yea that's what I use too.

**Comment by salewski** [comment]
+1 for HNDigital

**Comment by callegar** [comment]
As an advice for the time being, the practical way to tell others how to search for this software is to tell them to google for "digital+git" (comes out as 1st result). In general, when you look for software and you know it is opensource, `name+git` is a nice way to filter out a lot of noise.

**Comment by jxu** [comment]
HNDigital or Digisim - really anything would be an improvement
@hneemann

**Comment by dragostego** [comment]
It's truly impossible to search. even easy things become impossible meaning every question requires checking the documentation.

+1 arristottl or HNDigital

**Comment by 9EED** [comment]
a bump to this, it took me ages to find this repo after someone told me about this project

**Comment by jinnturtle** [comment]
I wonder if the project was intentionally meant to be less discoverable, if not - I vote for an alternative name also.

**Comment by LeoRC** [comment]
It doesn't help that not only is `digital` on its own a hopeless query, but `digital logic sim` is now associated with other applications ([Sebastian Lague's](https://sebastian.itch.io/digital-logic-sim) and [Lubin Pappalardo's](https://digital-logic-sim.com/)).

**Comment by shawty** [comment]
for a long time now I've been using "neeman digital", works pretty reliably for me, most times coming top in my search

<img width="1144" height="963" alt="Image" src="https://github.com/user-attachments/assets/e4dc29a2-067f-474d-aa3a-3353275006e1" />

Indecently, yes that video in the results does also make a point of being hard to find too.

https://www.youtube.com/watch?v=o7TQ-c1WNHg

**Comment by LeoRC** [comment]
> for a long time now I've been using "neeman digital", works pretty reliably for me, most times coming top in my search

True, searches like `neeman digital` seem to work well. But if you're a potential user who has just heard that 'Digital' is a good Logisim successor you normally won't know what the name of the project's leader is.

**Comment by brmarkus** [comment]
Let us all help spread the word then making "neeman digital" the "THE Digital Logic Designer" everyone needs to find!

**Comment by shawty** [comment]
> > for a long time now I've been using "neeman digital", works pretty reliably for me, most times coming top in my search
> 
> True, searches like `neeman digital` seem to work well. But if you're a potential user who has just heard that 'Digital' is a good Logisim successor you normally won't know what the name of the project's leader is.

Completely agree, but as you can see this issue has been open for a long time now, this is not the first time I've commented on it, while it doesn't help new users, it is never the less something we who know about it can use and spread the word about.

**Comment by alexthehurst** [comment]
"Neeman Digital" is actually a pretty good de-facto name to use for the program.

**Comment by EzraWolf** [comment]
Seems like the following are strongly preferred:
- Neeman Digital / NMDigital
- Helmut Digital / HMDigital

I'd argue to avoid "Digisim" as it is quite broad or "Digital Circuit Sim", aka., DCS, which is a very popular game.

If needed, I can design a logo stackup, just ping me.

**Comment by ellisgl** [comment]
How about Latidig?
